include/xrpl/basics/Number.h

```diff
@@ -304,29 +428,41 @@ operator/(Number const& x, Number const& y)
 inline constexpr Number
 Number::min() noexcept
 {
-    return Number{minMantissa, minExponent, unchecked{}};
+    return Number{range_.get().min, minExponent, unchecked{}};
 }
 
 inline constexpr Number
 Number::max() noexcept
 {
-    return Number{maxMantissa, maxExponent, unchecked{}};
+    return Number{range_.get().max, maxExponent, unchecked{}};
 }
 
 inline constexpr Number
 Number::lowest() noexcept
 {
-    return -Number{maxMantissa, maxExponent, unchecked{}};
+    return -Number{range_.get().max, maxExponent, unchecked{}};
```

Comment:
this function is no longer `constexpr` because it reads from `static thread_local range_`